File: src/ete-llorona/java/_2750.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();

        int[] nums = new int[n];

        for (int i = 0; i < n; i++) {
            nums[i] = scanner.nextInt();
        }

        int[] sorted_nums = sort(nums);

        for (int i = 0; i < n; i++) {
            System.out.println(sorted_nums[i]);
        }
    }

    public static int[] sort(int[] array) {

        for (int i = 0; i < array.length; i++) {
            for (int j = 0; j < (array.length - 1); j++) {

                int temp = 0;
                if (array[j] > array[j + 1]) {
                    temp = array[j + 1];
                    array[j + 1] = array[j];
                    array[j] = temp;
                }
            }
        }

        return array;
    }
}

File: src/ete-llorona/python/2750.py
n = int(input())
nums = [int(input()) for _ in range(n)]
# nums.sort()

for i in range(n):
    for j in range(n-1):
        if nums[j] > nums[j+1]:
            nums[j], nums[j+1] = nums[j+1], nums[j]

for num in nums:
    print(num)
